seaice_ecdr/temporal_composite_daily.py

```diff
@@ -817,7 +783,8 @@ def temporally_interpolated_ecdr_dataset_for_au_si_tbs(
     )
 
     # set non-conc values to -1
-    is_non_siconc = np.squeeze(tie_ds["cdr_conc"].data > 100)
+    # is_non_siconc = np.squeeze(tie_ds["cdr_conc"].data > 100)
+    is_non_siconc = tie_ds["cdr_conc"].data > 100
```

Comment:
```suggestion
    is_non_siconc = tie_ds["cdr_conc"].data > 1
```

This is fractional SIC.

Reply:
yep